src/main/java/database/service/JsonDatabaseService.java

```diff
@@ -44,6 +45,8 @@ public class JsonDatabaseService implements DatabaseService {
     static final String UNABLE_ACCESS_PROPERTY = "Unable to access property";
     static final String ENTITY_DOES_NOT_EXIST = "Entity with provided Id does not exist";
     static final String ID_PROVIDED_MANUALLY = "User cannot provide id manually. Ids are filled automatically.";
+    static final String INVALID_PARAMETER_VALUE = "Invalid parameter value. " +
+            "Limit value should not be less than 0, offset value should be more than 0 and less than 200";
```

Comment:
`offset` should be `>= 0`, `offset=0` is valid. Max value can be found only in `settings`.
The same for `limit`.
My suggestion here is to use stubs and replace stubs when throwing exception.
Something like:
`INVALID_PARAMETER_VALUE = "Invalid parameter value. Limit should be in(0..{MAX_LIMIT_VALUE}). Offset should be in(0..{MAX_OFFSET_VALUE}).";`

And throw like:
```
throw new InvalidParameterValueException(INVALID_PARAMETER_VALUE
    .replace("{MAX_LIMIT_VALUE}", settings.getLimit())
    .replace("{MAX_OFFSET_VALUE}", settings.getOffset()));
```